src/inject.js

```diff
@@ -47,7 +47,7 @@ const injectStyleTag = (cssContents /* : string */) => {
 
     if (styleTag.styleSheet) {
         // $FlowFixMe: legacy Internet Explorer compatibility
-        styleTag.styleSheet.cssText += cssContents;
+        styleTag.innerText += cssContents;
```

Comment:
Did you do any profiling to see what the performance impact this might be?

Also, I wonder if we should be using `insertRule()` instead?

https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule